tests/adapter/freshheads/CssAdapter.test.js

```diff
@@ -178,6 +178,25 @@ describe('FreshheadsCssAdapter', () => {
         });
     });
 
+    describe('With sass enabled', () => {
+        const config = {
+            sass: {
+                enabled: true,
+            },
+        };
+
+        const adapter = new FreshheadsCssAdapter(config);
+        const webpackConfig = {};
+
+        adapter.apply(webpackConfig, { env: 'dev' }, () => {});
+
+        // rule two is added when Sass Loader Adapter gets added
+        const lastTwoRules = webpackConfig.module.rules.slice(-2);
+
+        expect(lastTwoRules[0].test).toEqual(/\.s?css$/);
+        expect(lastTwoRules[1].test).toEqual(/\.scss$/);
```

Comment:
Moet je dit niet in een `it("shoud..", () => {})`-statement zetten?

Reply:
Hij test dit wel goed, je wil toch gewoon die value van die property vergelijken ?

Reply:
Het is zo ook niet echt de bedoeling ;)